.circleci/config.yml

```diff
@@ -798,13 +809,15 @@ workflows:
                 - dev
                 - stage
                 - prod
-      - deploy-reporter-tools:
+                - USAGOV-2278-v3
+      - deploy-reporter-dev:
```

Comment:
I suggest taking out the branch filter for deployment to dev. I've done that for the cms/www/waf/api-proxy deploy and, although it clutters up the pipeline options a little, it means we can deploy any branch to dev, which is nice.